package-res/resources/web/pentaho/common/propertiesPanel/ColorBox.js

```diff
@@ -14,66 +14,69 @@
  *
  * Copyright (c) 2012 Pentaho Corporation..  All rights reserved.
  */
-pen.define( ["dojo/_base/declare", "dojo/on", "dojo/_base/lang", "dijit/_Widget", "dijit/_TemplatedMixin", "dijit/_WidgetsInTemplateMixin", "pentaho/common/Dialog", "dijit/ColorPalette", "../ColorDialog", "dojo/dom-style"], function(declare, on, lang, Widget, TemplatedMixin, _WidgetsInTemplateMixin, Dialog, ColorPalette, ColorDialog, style) {
-    
-    return declare("pentaho.common.propertiesPanel.ColorBox", [
-        Widget,
-        TemplatedMixin,
-        _WidgetsInTemplateMixin
-    ],
-    {
-        value: "none",
-        okFunc: null,
-        templateString: "<div style='width:20px; height: 20px; background-color: none; cursor: pointer; border: 1px solid #808080;' dojoAttachPoint='colorBox'>&nbsp;</div>",
-        constructor:function (options) {
-            console.log('ColorBox.constructor');
-            this.inherited(arguments);
-        },
+define([
+    "dojo/_base/declare",
+    "dojo/on",
+    "dojo/_base/lang",
+    "dijit/_Widget",
+    "dijit/_TemplatedMixin",
+    "dijit/_WidgetsInTemplateMixin",
+    "pentaho/common/Dialog",
+    "dijit/ColorPalette",
+    "../ColorDialog",
+    "dojo/dom-style"
+], function(declare, on, lang, Widget, TemplatedMixin, _WidgetsInTemplateMixin, Dialog,
+            ColorPalette, ColorDialog, style) {
 
-        postCreate: function(){
-            console.log('ColorBox.postCreate');
-            style.set(this.colorBox,"background-color",this.value);
-            on(this.colorBox, "click", lang.hitch(this, "onClick"));
+    return declare(
+        "pentaho.common.propertiesPanel.ColorBox",
+        [
+            Widget,
+            TemplatedMixin,
+            _WidgetsInTemplateMixin
+        ],
+        {
+            value: "none",
+            okFunc: null,
+            templateString: "<div style='width:20px; height: 20px; background-color: none; cursor: pointer; " +
+                "border: 1px solid #808080;' dojoAttachPoint='colorBox'>&nbsp;</div>",
 
-        },
+            constructor: function(options) {
+                this.inherited(arguments);
+            },
 
-        onClick: function(){
-            console.log('ColorBox.onClick');
+            postCreate: function() {
+                style.set(this.colorBox,"background-color",this.value);
+                on(this.colorBox, "click", lang.hitch(this, "onClick"));
+            },
 
-            if( !ColorDialog.dialog ) {
-                ColorDialog.dialog = new ColorDialog();
-            }
-            var func = lang.hitch( this, function() { this.colorsChanged() } );
-            ColorDialog.dialog.registerOnSuccessCallback( func );
-            ColorDialog.dialog.show();
-        },
+            onClick: function() {
+                if(!ColorDialog.dialog) ColorDialog.dialog = new ColorDialog();
```

Comment:
It is a matter of style, of readability. Considered by some a bad practice, but, luckily, not by everyone.